main.tf

```
@@ -84,7 +85,8 @@ resource "aws_lb" "default" {
 }
 
 module "default_target_group_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source     = "cloudposse/null-label/aws"
```

```suggestion
  source     = "cloudposse/label/null"
```

For `terraform-null-label` module, the provider is `null`